cassandra-sink/src/main/java/org/springframework/cloud/stream/module/cassandra/sink/CassandraSinkApplication.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.cassandra.sink;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * @author Thomas Risberg
+ */
+@SpringBootApplication
+public class CassandraSinkApplication {
```

Comment:
It's interesting if we need this separate class at all.
Looks like it is the same in each module, which are the independent applications at all. 
So, no need to worry about the name of class.

We even could generate it... Or try to figure out some other way to bypass this boring copy/paste work...